### ISSUE TITLE
Make example of getPreselectedDates() and getDatesIndicators() in main page

### ISSUE THREAD

**Issue by moylir** [CleverPumpkin/CrunchyCalendar#44]
I think it would be convenient to make an example on how these functions could be implemented.

Indeed, I'm having problems to do a test  getDatesIndicators() function, though I'm trying to do it in java code as seen here: https://stackoverflow.com/questions/63298266/how-to-implement-kotlin-interface-in-java.

Thanks for the attention.

**Comment by Jordan160** [comment]
I agree with this, not enough information provided to get started with indicators.
